template.tpl

```diff
@@ -398,6 +469,11 @@ if (language === 'variable')
   scriptUrl += '&culture=' + encodeUriComponent(data.languageVariable);
 }
 
+// Add georegions to the script src
+if (geoRegionsString != "") {
```

Comment:
Should we also check for null etc?

Reply:
I believe it will always have a value, but I'll check :thumbsup:

Reply:
I see it's defined on line 342 as a string, so a check for null isn't necessary